Benchmarks/Benchmarks/Router/BinaryTrieRouterBenchmarks.swift

```diff
@@ -15,9 +15,9 @@
 import Benchmark
 @_spi(Internal) import Hummingbird
 
-func binaryTrieRouterBenchmarks() {
-    var trie: BinaryTrie<String>!
-    Benchmark("BinaryTrieRouter", configuration: .init(scalingFactor: .kilo)) { benchmark in
+func routerTrieRouterBenchmarks() {
+    var trie: RouterTrie<String>!
+    Benchmark("RouterTrieRouter", configuration: .init(scalingFactor: .kilo)) { benchmark in
```

Comment:
Can we make sure all the benchmark names in this file has the same prefix eg `RouterTrie:`